src/components/Contacts/ContactRow/ContactRow.tsx

```diff
@@ -60,83 +50,79 @@ export const ContactRow: React.FC<Props> = ({
     pledgeAmount,
     pledgeCurrency,
     pledgeFrequency,
+    pledgeReceived,
     primaryAddress,
     starred,
     status,
+    uncompletedTasksCount,
   } = contact;
 
   return (
-    <Box role="row" p={1}>
-      <Box display="flex" alignItems="center">
-        <Box padding="checkbox">
+    <ListItemButton focusRipple onClick={onClick} data-testid="rowButton">
+      <Hidden xsUp={contactDetailsOpen}>
+        <ListItemIcon>
           <Checkbox
             checked={isChecked}
-            color="default"
+            color="secondary"
+            onClick={(event) => event.stopPropagation()}
             onChange={() => onContactCheckToggle(contact.id)}
             value={isChecked}
           />
-        </Box>
-        <ContactRowButton data-testid="rowButton" onClick={onClick}>
-          <ContactTextWrap>
-            <ContactText
-              style={{
-                fontSize: '16px',
-                letterSpacing: '0.15px',
-              }}
-            >
-              {name}
-            </ContactText>
-            <ContactText>{primaryAddress?.street || ''}</ContactText>
-          </ContactTextWrap>
-
-          <Hidden smDown>
-            <Box
-              style={{
-                display: 'inline-block',
-                margin: theme.spacing(1),
-              }}
-            >
-              <CelebrationIcons contact={contact} />
-            </Box>
-          </Hidden>
-
-          <Box
-            style={{
-              display: 'inline-block',
-              flexBasis: 0,
-              margin: theme.spacing(1),
-            }}
-          >
-            <GiftStatus lateAt={lateAt ?? undefined} />
-          </Box>
-
-          <Hidden mdDown>
-            <Box
-              style={{
-                display: 'inline-block',
-                flexGrow: 4,
-                flexBasis: 0,
-                margin: theme.spacing(1),
-              }}
-            >
-              <ContactText>{status ?? ''}</ContactText>
-              <ContactText>
-                {pledgeAmount
-                  ? pledgeCurrency
-                    ? `${pledgeAmount} ${pledgeCurrency}`
-                    : pledgeAmount
-                  : ''}{' '}
-                {pledgeFrequency ?? ''}
-              </ContactText>
-            </Box>
-          </Hidden>
+        </ListItemIcon>
+      </Hidden>
+      <Grid container alignItems="center">
+        <Grid item xs={6}>
```

Comment:
I really like the ```Grid``` component! Makes aligning and spacing super simple 🙂 Can get a bit confusing when you get into deep nesting of grids, but it beats having to write a bunch of CSS to achieve the same thing.